src/desktop/apps/home/templates/index.jade

```diff
@@ -36,8 +36,18 @@ block body
     #home-layout-container.responsive-layout-container
       .main-layout-container: #home-body
         include featured_links
-        unless user
+
+    if sd.COLLECTIONS_HUBS_DATA && sd.COLLECTIONS_HUBS_DATA.length
+      #home-layout-container.responsive-layout-container
+        .main-layout-container: #home-body
+          h2.home-featured-header Featured categories
+          != stitch.components.CollectionsHubsHomepageNav({collectionsHubs: sd.COLLECTIONS_HUBS_DATA})
+
+    unless user
```

Comment:
I had to break up the existing `#home-layout-container > .main-layout-container`, to wedge the hubs in between the `featured_links` and `browse` sections.